### PR TITLE
install perf in development builds

### PR DESCRIPTION
**Pull request by HiassofT**
perf is an excellent profiling tool and it's handy to have it available in devbuilds

It's only installed if performance events are enabled in the kernel config (CONFIG_PERF_EVENTS), which means it'll currently only be built for RPi and Generic

### CODE REVIEW

File: packages/linux/package.mk [hunk 1]
```diff
@@ -29,6 +29,32 @@ PKG_SHORTDESC="linux26: The Linux kernel 2.6 precompiled kernel binary image and
 PKG_LONGDESC="This package contains a precompiled kernel image and the modules."
 PKG_IS_KERNEL_PKG="yes"
 
+PKG_CFG_FILE="$PKG_NAME.${TARGET_PATCH_ARCH:-$TARGET_ARCH}.conf"
```
Comment (chewitt) [inline packages/linux/package.mk]: this block can be removed/simplified after #2432 no?

Comment (MilhouseVH) [reply packages/linux/package.mk]: No, this boiler plate hasn't been simplified (yet) - we could add a helper function `get_kernel_config_file()` that will simplify/centralise this. I'll look at it for a separate PR.

Comment (MilhouseVH) [reply packages/linux/package.mk]: Added #2485

**Comment by lrusak** [comment]
This broke mesa for me on iMX6 because of libunwind

We should probably go through all the packages and see if any packages have a configure switch for libunwind

**Pull request merged**